Assets/QA/Input_Test/Scripts/Input/DualShockISX.cs

```diff
@@ -19,19 +19,19 @@ void Start()
         //m_discreteButtonAction.canceled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
         //m_discreteButtonAction.Enable();
 
-        m_buttonAction = new InputAction(name: "DualShockButtonAction", binding: "*DualShock*/<button>");
+        m_buttonAction = new InputAction(name: "DualShockButtonAction", binding: "*DualShock*/<button>") { passThrough = true }; ;
         m_buttonAction.performed += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
-        m_buttonAction.canceled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
+        //m_buttonAction.cancelled += callbackContext => OnControllerButtonPress(callbackContext.control as ButtonControl, isPS: true);
```

Comment:
Why did you comment this (and the ones below) out? This API has been renamed to match the preferred US spelling, it should work using "canceled".